hack/update-all.sh

```diff
@@ -90,4 +90,9 @@ do
 	fi
 done
 
-echo -e "${color_green}Update scripts completed successfully${color_norm}"
+if kube::util::ensure_clean_working_dir &>/dev/null; then
+    exec "${KUBE_ROOT}/hack/update-all-staging.sh" "$@"
```

Comment:
With https://github.com/kubernetes/kubernetes/pull/42728#issuecomment-285091668 in mind I tend to prefer removing the clean-repo requirement from the problematic `hack/update-*` scripts. The user experience is pretty bad right now. There is not much sense in `hack/update-all-*` if you end up with committing after each sub-script and then run the next manually.